easybuild/easyblocks/b/bamtools.py

```diff
@@ -58,10 +58,22 @@ def sanity_check_step(self):
         sharedlib_ext = get_shared_lib_ext()
 
         custom_paths = {
-            'files': ["bin/bamtools", "include/shared/bamtools_global.h", "lib/libbamtools.a",
-                      "lib/libbamtools.%s" % sharedlib_ext, "lib/libbamtools-utils.%s" % sharedlib_ext,
-                      "lib/libjsoncpp.%s" % sharedlib_ext],
-            'dirs': ["include/api", "docs"]
+            'files': [
+                "bin/bamtools",
+                "include/shared/bamtools_global.h",
+                "lib/libbamtools.a",
+                "lib/libbamtools.%s" % sharedlib_ext],
```

Comment:
Weird, it's there: http://legacy.python.org/dev/peps/pep-0008/#indentation (at the end)

Yeah, style fixes are annoying but with a good editor, they are done properly by default?